### PR TITLE
Manage client roles for users

### PR DESCRIPTION
**Pull request by chrnin**
Hi,

I just added 2 functions to the gocloak type and implemented them in client.go.
I tested it against keycloak 6.0.1 and it seems functional.

Have a good day :)

Christophe.

### CODE REVIEW

**Comment by chrnin** [comment]
I realize I didn't wrote any tests ... Maybe I could try that tomorrow.

File: gocloak.go [hunk 1]
```diff
@@ -128,6 +128,12 @@ type GoCloak interface {
 	// AddRealmRoleComposite adds roles as composite
 	DeleteRealmRoleComposite(token string, realm string, roleName string, roles []Role) error
 
+	// *** Client Roles ***
```
Comment (SVilgelm) [inline gocloak.go]: Please add this functions to `README.md`

Comment (chrnin) [reply gocloak.go]: I just updated README.md, I'll try to write test, I don't know if I'll have time today.

**Comment by SVilgelm** [comment]
Closing it in favor of #91

**Pull request closed without merging**